public/components/trace_analytics/components/traces/trace_view.tsx

```diff
@@ -65,105 +66,117 @@ export function TraceView(props: TraceViewProps) {
     );
   };
 
-  const renderOverview = (fields: any) => {
+  const [fields, setFields] = useState({});
+  const [serviceBreakdownData, setServiceBreakdownData] = useState([]);
+  const [payloadData, setPayloadData] = useState('');
+  const [colorMap, setColorMap] = useState({});
+  const [ganttData, setGanttData] = useState<{ gantt: any[]; table: any[]; ganttMaxX: number }>({
+    gantt: [],
+    table: [],
+    ganttMaxX: 0,
+  });
+  const [serviceMap, setServiceMap] = useState<ServiceObject>({});
+  const [traceFilteredServiceMap, setTraceFilteredServiceMap] = useState<ServiceObject>({});
+  const [serviceMapIdSelected, setServiceMapIdSelected] = useState<
+    'latency' | 'error_rate' | 'throughput'
+  >('latency');
+  const [isServicesDataLoading, setIsServicesDataLoading] = useState(false);
+  const [isTraceOverViewLoading, setIsTraceOverViewLoading] = useState(false);
+  const [isTracePayloadLoading, setTracePayloadLoading] = useState(false);
+  const [isServicesPieChartLoading, setIsServicesPieChartLoading] = useState(false);
+
+  const renderOverview = (overviewFields: any) => {
     return (
       <EuiPanel>
         <PanelTitle title="Overview" />
-        <EuiHorizontalRule margin="m" />
-        <EuiFlexGroup>
-          <EuiFlexItem>
-            <EuiFlexGroup direction="column">
-              <EuiFlexItem grow={false}>
-                <EuiText className="overview-title">Trace ID</EuiText>
-                {fields.trace_id && (
-                  <EuiFlexGroup gutterSize="s" alignItems="center">
+        {isTraceOverViewLoading ? (
+          <div>
+            <EuiLoadingContent lines={4} />
+          </div>
+        ) : (
+          <>
+            <EuiHorizontalRule margin="m" />
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <EuiFlexGroup direction="column">
+                  <EuiFlexItem grow={false}>
+                    <EuiText className="overview-title">Trace ID</EuiText>
+                    {overviewFields.trace_id && (
+                      <EuiFlexGroup gutterSize="s" alignItems="center">
+                        <EuiFlexItem grow={false}>
+                          <EuiText size="s" className="overview-content">
+                            {overviewFields.trace_id}
+                          </EuiText>
+                        </EuiFlexItem>
+                        <EuiFlexItem grow={false}>
+                          <EuiCopy textToCopy={overviewFields.trace_id}>
+                            {(copy) => (
+                              <EuiSmallButtonIcon
+                                aria-label="Copy trace id"
+                                iconType="copyClipboard"
+                                onClick={copy}
+                              >
+                                Click txo copy
```

Comment:
Typo "Click to copy"

Reply:
Great catch! Thanks looks like this was there for many releases